libs3/src/object.c

```diff
@@ -497,8 +502,163 @@ void S3_restore_object(S3BucketContext* bucketContext,
 		restoreObjectCallback,           // fromS3Callback
 		restoreObjectCompleteCallback,   // completeCallback
 		data,                            // callbackData
-		timeoutMs                        // timeoutMs
+		timeoutMs,                       // timeoutMs
+		0                                // xAmzObjectAttributes
 	};
 
 	request_perform(&params, requestContext);
 }
+
+// get object attributes ----------------------------------------------------------------
```

Comment:
Is the dotted lines styling a convention seen in other source files of the libs3 library?

Reply:
Yes.  All of the requests (get object, delete object, etc.) follow this pattern of dotted lines.